### DISCUSSION TITLE
Enquiry about using interface diffusion coefficient calculated by DFT in phase field

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#19545] [Q&A Modules: Phase field] [answered]
Dear,

I have a question about phase field that bothers me a lot.

Usually in phase field, people use an interpolation function to give the diffusion coefficient at the interface(interpolate the coefficient between phase A and phase B). But if through DFT calculation, I can clearly compute the diffusion coefficient at the interface, is there a way I can use the calculated coefficient instead of the interpolation one? If there is a way, then the physical property at the interface may be more meaningful. Also, we can explore more nanoscale factors’ influence on the microstructure evolution through coupling between DFT and PF. 
                
Any response is highly appreciated!

Best wishes,
zhigang

**Comment by hugary1995** [comment, marked as answer]
Are you talking about the "Multi-scale simulation" animation on the [gallery page?](https://mooseframework.inl.gov/gallery.html) MOOSE uses the [MultiApp](https://mooseframework.inl.gov/syntax/MultiApps/index.html) system to achieve this.

**Reply by laagesen** [reply]
You could create an interpolation function that is non-zero only in the interface, then multiply that by the diffusion coefficient. For example, say you have a two-phase system with represented by an order parameter eta that is 1 in the alpha phase and 0 in the beta phase. As you said, normally the diffusivity between the two phases would be represented by function like D_alpha * h + D_beta * (1-h), where h is an interpolation function. If the diffusivity at the interface is much larger than in the bulk, you could make the total diffusivity as D_alpha * h + D_beta * (1-h) + D_int * eta^2 * (1-eta)^2, so that the contribution from interfacial diffusivity is only non-zero at the interface.
